apps/experiments/models.py

```diff
@@ -412,6 +512,16 @@ def tools_enabled(self):
     def event_triggers(self):
         return [*self.timeout_triggers.all(), *self.static_triggers.all()]
 
+    @property
+    def has_versions(self):
+        return self.versions.count() > 0
+
+    @property
+    def version_display(self) -> str:
+        if self.is_working_version:
+            return None
```

Comment:
I think this will results in displaying `'None'`:
```suggestion
            return ""
```

Reply:
Thanks. I wil incorporate all these changes into a new PR that will usher in the logic updates